Add `osu!catch World Cup 2025: Round of 32 Recap` news post (#13406)
ppy/osu-wiki: news/2025/2025-06-20-osucatch-world-cup-2025-round-of-32-recap.md

This osu!catch World Cup promises to be one for the ages, as many teams and players are returning and aiming for the top. It's also notably the edition with the most countries playing, with a grand total of 44 who competed for a spot in this Round of 32. The ones who qualified got to play this past week, so let's look at how it went.

## Mappool recap

<style>
    .osu-md--news .audio-player {
        max-width: inherit;
    }
</style>

**[Download the pool here! (99 MB)](https://osu.ppy.sh/beatmaps/packs/P288)**

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?autoplay=false&parent=osu.ppy.sh&time=1h4m23s&video=2480836532" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

![](/wiki/shared/news/2025-06-20-osucatch-world-cup-2025-round-of-32-recap/cwc-2025-ro32-mappool.png)

The recent osu!catch World Cups have had a trend of mappools getting noticeably harder every year, and this edition is no exception. While still approachable for every team that qualified, some maps left even the most seasoned teams struggling to get FCs.

> One such example would be the second No Mod map, [Island of Orchids](https://osu.ppy.sh/beatmapsets/2384624#fruits/5156307) mapped by [Malai](https://osu.ppy.sh/users/4863096) (NM2). Featuring very inconsistent and technical stream spacings and shapes, it ended up being one of the toughest maps this week and a testament to the rising difficulty of these mappools. The song also comes from friend of the community and Featured Artist [WangleLine](https://osu.ppy.sh/users/7994819), which we always love to see.

Reviewer: "friend of the community" and "community favourite" aren't the same thing <.<
sounds ok as is

Right from the get-go, Belgium looked solid. On China's very first pick of the match on [MM2](https://osu.ppy.sh/beatmapsets/2384646#fruits/5156338), they managed to score a breakpoint by only 7,000 points. After a convincing win on [NM2](https://osu.ppy.sh/beatmapsets/2384624#fruits/5156307) from Belgium, featuring an impressive FC from ::{ flag=BE }:: [Jezse](https://osu.ppy.sh/users/11390632), the close points would come back. On China's [HD1](https://osu.ppy.sh/beatmapsets/2384632#fruits/5156316) pick, they once again missed out on a point, this time by less than 2,000 points, with one SS on each side, ::{ flag=BE }:: [ElSkeffe](https://osu.ppy.sh/users/6283136) for Belgium and ::{ flag=CN }:: [dazzulay](https://osu.ppy.sh/users/1948043) for China, respectively. And again, on [DT3](https://osu.ppy.sh/beatmapsets/2384641#fruits/5156332), the score difference would be of only 14,000 for Belgium.

> Despite their best efforts, China found themselves down 4–0, and Belgium only had to win one point to close out the match. However, the momentum would shift on their next pick, [NM3](https://osu.ppy.sh/beatmapsets/2384639#fruits/5156330), with only 3 misses across their players. They weren't about to go down without a fight, and continued their comeback on Belgium's next pick, [NM1](https://osu.ppy.sh/community/matches/118392467), with FCs on each side for ::{ flag=CN }:: [Cyphos](https://osu.ppy.sh/users/9740698), ::{ flag=BE }:: [Jezse](https://osu.ppy.sh/users/11390632), and ::{ flag=BE }:: [[ Ajia ]](https://osu.ppy.sh/users/17578266). It was once again a very close point, but this time it was in their favour. Their consistency and relentlessness were being rewarded, and they carried on to their [NM4](https://osu.ppy.sh/beatmapsets/2384633#fruits/5156317) pick, which they successfully won.

Reviewer: ```suggestion
Despite their best efforts, Team China found itself down 4–0, and Belgium only had to win one point to close out the match. However, the momentum would shift on their next pick, [NM3](https://osu.ppy.sh/beatmapsets/2384639#fruits/5156330), with only 3 misses across their players. They weren't about to go down without a fight, and continued their comeback on Belgium's next pick, [NM1](https://osu.ppy.sh/community/matches/118392467), with FCs on each side for ::{ flag=CN }:: [Cyphos](https://osu.ppy.sh/users/9740698), ::{ flag=BE }:: [Jezse](https://osu.ppy.sh/users/11390632), and ::{ flag=BE }:: [[ Ajia ]](https://osu.ppy.sh/users/17578266). It was once again a very close point, but this time it was in China's favour. Their consistency and relentlessness were being rewarded, and they carried on to their [NM4](https://osu.ppy.sh/beatmapsets/2384633#fruits/5156317) pick, which they successfully won.
```

> However, unlike Singapore who managed to get to a tiebreaker from a big score deficit on their own match, it wouldn't be the case for China as Belgium managed to close out the match before that with a great showcase of consistency on [HR1](https://osu.ppy.sh/beatmapsets/2384631#fruits/5156315), for a final score of 5–3. The whole match was very intense with many close points, and it's always very hype to see the losing team make a comeback, even if it doesn't always end up on an upset. Go watch the match if you haven't!

Reviewer: ```suggestion
However, unlike the Singaporeans who managed to get to a tiebreaker from a big score deficit on their own match, it wouldn't be the case for China as Belgium managed to close out the match before that with a great showcase of consistency on [HR1](https://osu.ppy.sh/beatmapsets/2384631#fruits/5156315), for a final score of 5–3. The whole match was very intense with many close points, and it's always very hype to see the losing team make a comeback, even if it doesn't always end up on an upset. Go watch the match if you haven't!
```

---

> The Round of 16 is coming up this weekend, and the first eliminations will happen. Make sure to tune in to the stream at [osulive](https://www.twitch.tv/osulive) and [osulive_2](https://www.twitch.tv/osulive_2) for a weekend full of exciting matches. You can find match listings on the [CWC wiki page](/wiki/Tournaments/CWC/2025) with detailed statistics for each round, such as [the ones for this Round of 32](https://docs.google.com/spreadsheets/d/1pU0uX-VahJp5zig_tg_dudJ_b4LnHe_Q8irzdnMtFw8?rm=minimal).

Reviewer: ```suggestion
The Round of 16 is coming up this weekend, and the first eliminations will happen. Make sure to tune in to the stream at [osulive](https://www.twitch.tv/osulive) and [osulive_2](https://www.twitch.tv/osulive_2) for a weekend full of exciting matches, the first one being Iceland vs. Singapore at 14:00 UTC this Saturday. You can find the rest of the schedule on the [CWC wiki page](/wiki/Tournaments/CWC/2025) complete with detailed statistics for each round, such as [the ones for this Round of 32](https://docs.google.com/spreadsheets/d/1pU0uX-VahJp5zig_tg_dudJ_b4LnHe_Q8irzdnMtFw8?rm=minimal).
```
